tests/pjit_test.py

```diff
@@ -1295,18 +1295,88 @@ def f(x):
         pjit_f(jnp.array([1, 2, 3]))
 
   def test_pretty_print(self):
-    f = pjit(lambda x: x)
+    f = pjit(lambda x: x**2)
+    g = pjit(lambda x: f(x) + f(x))
     x = jnp.array([4.2], dtype=jnp.float32)
-    jaxpr = jax.make_jaxpr(f)(x)
+    jaxpr = jax.make_jaxpr(g)(x)
+    self.assertEqual(
+        jaxpr.pretty_print(),
+        textwrap.dedent("""
+            let <lambda> = { lambda ; a:f32[1]. let b:f32[1] = integer_pow[y=2] a in (b,) } in
```

Comment:
Maybe strip < and >.

Reply:
Done.

Note that this makes `lambda` a soft keyword.

Reply:
Well, I was just thinking "it's a name like any other name", and "you'll uniquify it like any other name".